src/reduced_basis/rb_parameters.C

```diff
@@ -150,30 +150,35 @@ void RBParameters::push_back_extra_value(const std::string & param_name, Real va
 Real RBParameters::get_extra_value(const std::string & param_name) const
 {
   // Same as get_value(param_name) but for the map of extra parameters
-  const auto & vec = libmesh_map_find(_extra_parameters, param_name);
-  libmesh_error_msg_if(vec.size() == 0, "Error getting value for extra parameter " << param_name);
-  return vec[0];
+  const auto & sample_vec = libmesh_map_find(_extra_parameters, param_name);
+  libmesh_error_msg_if(sample_vec.size() != 1,
+    "Requesting value for extra parameter " << param_name << ", but parameter contains multiple samples.");
+  return sample_vec[0];
 }
 
 Real RBParameters::get_extra_value(const std::string & param_name, const Real & default_val) const
 {
   // same as get_value(param_name, default_val) but for the map of extra parameters
   auto it = _extra_parameters.find(param_name);
-  return ((it != _extra_parameters.end() && it->second.size() != 0) ? it->second[0] : default_val);
+  if(it==_parameters.end())
```

Comment:
Space after `if` in if-statements and preferably around comparison operators, so:
```
if (it == _parameters.end())
```
We do have some [source code formatting conventions](https://github.com/libMesh/libmesh/wiki/Source-code-formatting-conventions) but nothing that automatically enforces them... I can push a commit fixing stuff like this if you want.